community/dbms/src/main/java/org/neo4j/commandline/dbms/CsvImporter.java

```diff
@@ -0,0 +1,82 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.commandline.dbms;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.neo4j.kernel.configuration.Config;
+import org.neo4j.tooling.ImportTool;
+
+import static org.neo4j.dbms.DatabaseManagementSystemSettings.database_path;
+
+class CsvImporter implements Importer
+{
+    public static String description()
+    {
+        return "--mode=csv Import a database from a collection of CSV files.\n" +
+                "--nodes[:Label1:Label2]=\"<file1>,<file2>,...\"\n" +
+                "        Node CSV header and data. Multiple files will be logically seen as\n" +
+                "        one big file from the perspective of the importer. The first line\n" +
+                "        must contain the header. Multiple data sources like these can be\n" +
+                "        specified in one import, where each data source has its own header.\n" +
+                "        Note that file groups must be enclosed in quotation marks.\n" +
+                "--relationships[:RELATIONSHIP_TYPE] \"<file1>,<file2>,...\"\n" +
+                "        Relationship CSV header and data. Multiple files will be logically\n" +
+                "        seen as one big file from the perspective of the importer. The first\n" +
+                "        line must contain the header. Multiple data sources like these can be\n" +
+                "        specified in one import, where each data source has its own header.\n" +
+                "        Note that file groups must be enclosed in quotation marks.\n" +
+                "--input-encoding <character set>\n" +
+                "        Character set that input data is encoded in. Provided value must be\n" +
+                "        one out of the available character sets in the JVM, as provided by\n" +
+                "        Charset#availableCharsets(). If no input encoding is provided, the\n" +
+                "        default character set of the JVM will be used.\n" +
+                "--id-type <id-type>\n" +
+                "        One out of [STRING, INTEGER, ACTUAL] and specifies how ids in\n" +
+                "        node/relationship input files are treated.\n" +
+                "            STRING: arbitrary strings for identifying nodes.\n" +
+                "            INTEGER: arbitrary integer values for identifying nodes.\n" +
+                "            ACTUAL: (advanced) actual node ids. The default option is STRING.\n" +
+                "        Default value: STRING\n";
+    }
```

Comment:
I think you need a preamble here saying what it actually does. You can take text from the old man page or from the docs.